.github/actions/build-sign-and-package-plugin/action.yml

```diff
@@ -2,31 +2,35 @@ name: Build, sign, and package plugin
 description: Build, sign, and package plugin
 inputs:
   plugin_version_number:
-    description: "The version number of the plugin"
+    description: |
+      The version number of the plugin. NOTE: this action will chop off the leading "v" to use
+      it as the official plugin version.
     required: true
-  grafana_access_policy_token:
-    description: "The Grafana access policy token used to sign the plugin"
-    required: true
-  working_directory:
-    description: "The working directory of the plugin"
-    required: true
-  is_enterprise:
-    description: "Whether the plugin is an enterprise build or not"
-    required: false
-    default: "false"
 outputs:
   artifact_filename:
     description: "The filename of the plugin artifact"
     value: ${{ steps.artifact-filename.outputs.filename }}
 runs:
   using: "composite"
   steps:
+    # This will fetch the secret keys from vault and set them as environment variables for subsequent steps
+    - name: Get Vault secrets
+      uses: grafana/shared-workflows/actions/get-vault-secrets@main
+      with:
+        repo_secrets: |
+          GRAFANA_ACCESS_POLICY_TOKEN=github_actions:cloud-access-policy-token
+    - name: Determine official plugin version
+      id: plugin-version
+      shell: bash
+      run: |
+        # VERY IMPORTANT: chop off the "v".. this tells the oncall plugin that this is an OSS build
+        PLUGIN_VERSION="$(echo ${{ inputs.plugin_version_number }} | sed 's/^v//')"
```

Comment:
this is what will fix https://github.com/grafana/oncall/issues/5028.

The version number that is transformed on this line is then used [here](https://github.com/grafana/oncall/pull/5051/files#diff-6200ac63c387869ca55bc34d1b343a96c667daef428602f910d696836cb09f84R33) to determine the artifact filename, which is then used [here](https://github.com/grafana/oncall/pull/5051/files#diff-e95a5d3f03a1351728732657b6b150cfbbd9a9724b387226b1f99f079b1954b0R60) to publish the artifact to GCS, and finally that GCS URL is used [here](https://github.com/grafana/oncall/pull/5051/files#diff-e95a5d3f03a1351728732657b6b150cfbbd9a9724b387226b1f99f079b1954b0R63) to publish the version with gcom